src/site/stages/build/drupal/graphql/taxonomy-fragments/GetTaxonomies.graphql.js

```diff
@@ -0,0 +1,21 @@
+module.exports = `
+  allTaxonomies: taxonomyTermQuery(limit: 1000) {
+    entities {
+      entityBundle
+      ... on TaxonomyTermAudienceNonBeneficiaries {
+        fieldAudienceRsHomepage
+        name
+        path {
+          alias
+        }
+      }
+      ... on TaxonomyTermAudienceBeneficiaries {
+        fieldAudienceRsHomepage
+        name
+        path {
+          alias
+        }
+      }
+    }
+  }
+`;
```

Comment:
This new GraphQL query grabs all of the new CMS's taxonomies. We don't need all of them - we only need "audience tags." I can't get that GraphQL filter to work though, so the FE does the filter instead.

Reply:
Uggh that's frustrating

Reply:
Thanks for getting me started with that original `taxonomyTermQuery`!

Reply:
Of course! 🙂